### DISCUSSION TITLE
Erreur dans la création d'algorithmes

### DISCUSSION THREAD

**Discussion by NilsInfo** [danymat/INSAnonym-utils#92] [Q&A, Bug report & Support]
J'essaye de créer un pseudo-algorithme pour tester la création d'algorithmes mais le return de cet algorithme n'est pas pris en compte.
# L'algo : (clusterizer.py)
```
def main(df, options):
    newDf = df[["latitude","longitude"]]
    return newDf
```
# La partie algorithme du parser.cfg
```
"algorithms": [
    { "name": "clusterizer", "options": { } }
  ],
```
# Le script:
```
from insanonym_utils import models,runner
model = models.FileConfigModel.parse_file('parser.cfg')
r = runner.Runner(model)
print(r.dataframe)
r.execute()
print(r.dataframe)
```
Les 2 print(r.dataframe) renvoient la même chose 
```
     id                date  latitude  longitude
0      1 2015-03-04 00:35:16  4.870147  45.772140
1      1 2015-03-04 00:35:48  4.870218  45.772095
2      1 2015-03-04 00:35:49  4.870210  45.772072
3      1 2015-03-04 00:35:50  4.870210  45.772072
4      1 2015-03-04 00:35:52  4.870210  45.772072
...   ..  
```
Alors que le newDf est égal à :
```
latitude  longitude
0     4.870147  45.772140
1     4.870218  45.772095
2     4.870210  45.772072
3     4.870210  45.772072
4     4.870210  45.772072
...        ...        ...
```
Les fichiers sont tous au même niveau du repo.
Avez-vous une idée de pourquoi l'algorithme ne modifie rien ? Si je prends un algo préfait, le second print est modifié en fonction de l'algo

**Comment by beng-git** [comment]
Bonjour, 

est-ce le même problème que : https://github.com/danymat/INSAnonym-utils/issues/89#issue-1050040123 ?
